.github/workflows/test_openvino_basic.yml

```
@@ -24,7 +24,7 @@ jobs:
       matrix:
         # Testing lower and upper bound of supported Python versions
         # This also ensures that the test fails if dependencies break for Python 3.7
-        python-version: ["3.7", "3.10"]
+        python-version: ["3.10"]
```

```suggestion
        python-version: ["3.8", "3.11"]
```
This test is pretty quick and I find it useful to continue testing upper and lower bounds.